circle.yml

```diff
@@ -0,0 +1,33 @@
+#
+# Build configuration for Circle CI
+#
+
+machine:
+    timezone: Asia/Kolkata
+    java:
+        version: oraclejdk8
+    environment:
+        ANDROID_HOME: /usr/local/android-sdk-linux
+
+
+dependencies:
+    override:
+            - chmod +x gradlew
+            - ./gradlew dependencies
+    pre:
+        - echo y | android update sdk --no-ui --all --filter tools,platform-tools,extra-google-m2repository,extra-google-google_play_services,extra-android-support,extra-android-m2repository,android-25
+        - echo y | android update sdk --no-ui --all --filter build-tools-25.0.3
+    cache_directories:
+        - /usr/local/android-sdk-linux/tools
+        - /usr/local/android-sdk-linux/build-tools/25.0.3
+test:
+  pre:
+    - emulator -avd circleci-android22 -no-window:
+        background: true
+        parallel: true
+  override:
+    - ./gradlew clean assembleDebug
+    - circle-android wait-for-boot:
```

Comment:
why wait for boot when no connectedCheck ? 

Like please can you read your code before you submit.
Do not care to submit code that you have no idea about.